make_release.py

```diff
@@ -39,14 +39,14 @@ def fill(s):
 
     # TODO: this should be echocall(redirect_stdout=False)
     print(fill('Updating larrayenv metapackage to version {version}'))
-    # - fixed versions of numpy and pytables because of incompatibilities
-    #   between pytables <= 3.4 and numpy >= 1.16
-    #   Didn't set pytables ==3.5 because it is not available for Python 3.5
+    # - needs pytables =>3.5 to:
+    #      * get root._v_attrs available (used in core/metadata.py)
```

Comment:
hu? is it True? There ought to be an alternate syntax which worked with earlier versions of pytables, as this makes larray much harder to install. There is nothing in pytables 3.5 changelog related to that!?

Reply:
I remember that I had to update the version of pytables to 3.4.X of one of our users. I am think he had a 3.3.X version but I am not totally sure.